driver-async/src/test/functional/com/mongodb/async/client/TransactionsTest.java

```diff
@@ -130,7 +135,22 @@ public void setUp() {
 
         BsonDocument clientOptions = definition.getDocument("clientOptions", new BsonDocument());
 
-        mongoClient = MongoClients.create(Fixture.getMongoClientBuilderFromConnectionString()
+        ConnectionString connectionString = getConnectionString();
+        if (this.filename.equals("pin-mongos.java")) {
+            connectionString = getMultiMongosConnectionString();
```

Comment:
Do tests all pass if we use the multiMongos connection string for all tests?  They should, and I think we'd get slightly better test coverage if we did use that, since as we saw the pin-mongos tests don't have a lot of assertions in them.

Reply:
Ping on this one.